docs/extensibility/walkthrough-publishing-a-visual-studio-extension-via-command-line.md

```diff
@@ -17,7 +17,7 @@ ms.workload:
 ---
 # Walkthrough: Publishing a Visual Studio extension via command line
 
-This walkthrough shows you how to publish a Visual Studio extension to the Visual Studio Marketplace using the command line. When you add your extension to the Marketplace, developers can use the **Extensions and Updates** dialog to browse there for new and updated extensions.
+This walkthrough shows you how to publish a Visual Studio extension to the Visual Studio Marketplace using the command line. When you add your extension to the Marketplace, developers can use the [**Extensions and Updates**](https://docs.microsoft.com/visualstudio/ide/finding-and-using-visual-studio-extensions?view=vs-2017) dialog to browse there for new and updated extensions.
```

Comment:
For document relative links please use the following format:
[**Extensions and Updates**](../ide/finding-and-using-visual-studio-extensions.md).
You can find examples of this relative link format within the repo.

Reply:
Thanks @gregvanl, I've added a new commit with the change.